### PR TITLE
Refactor config service

### PR DESCRIPTION
**Pull request by titkov49**
Config Service gets config url from meta with name `x-config`

### CODE REVIEW

**Comment by titkov49** [comment]
@ateska 
At this point I handled only extracting meta and it's content from header. I tried `import` and `require()` to get `./my-config.json` but faced some issues with that. And I just understood that content inside meta element should be url with which I should download configs with axios, am I right?

File: src/config/ConfigService.js [hunk 1]
```diff
@@ -42,6 +42,9 @@ export default class ConfigService extends Service {
 	initialize() {
 		let config_url = this.App.Config.get('CONFIG_PATH');
 
+		const xConfig = document.getElementsByName('x-config')[0];
```
Comment (ateska) [inline src/config/ConfigService.js:45]: `X-Config`

**Review by ateska — CHANGES_REQUESTED** [review]
Good!

**Comment by ateska** [comment]
@titkov49 the goal is something like:

```
let config_url = document.getElementsByName('X-Config')[0];
```

The download of the config is already implemented.

File: src/config/ConfigService.js (hunk 1 above)
Comment (titkov49) [reply src/config/ConfigService.js:45]: Dashes are not allowed in naming in javascript (parser assume that it is subtract operator). I can use underscore instead.

**Comment by titkov49** [comment]
@ateska I removed old code for now. But maybe I should keep it in order to keep support of old configuration?

File: src/config/ConfigService.js (hunk 1 above)
Comment (ateska) [reply src/config/ConfigService.js]: I meant:
```
const xConfig = document.getElementsByName('X-Config')[0];
```

File: src/config/ConfigService.js [hunk 2]
```diff
@@ -40,18 +40,13 @@ export default class ConfigService extends Service {
 
 
 	initialize() {
-		let config_url = this.App.Config.get('CONFIG_PATH');
-
-		// Provide backward compatibility with CONFIG_URL
-		if (this.App.Config.get('CONFIG_URL')) {
-			config_url = this.App.Config.get('CONFIG_URL');
-		}
+		const X_Config = document.getElementsByName('x-config')[0].content;
```
Comment (ateska) [inline src/config/ConfigService.js:43]:
```
const dynamic_config_url = document.getElementsByName('x-config')[0].content;
```

Local variables are always in lowercase + underscores.

**Review by ateska — CHANGES_REQUESTED** [review]
Styling remarks, then mergeable.

**Comment by ateska** [comment]
> @ateska I removed old code for now. But maybe I should keep it in order to keep support of old configuration?

No.

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@titkov49 Add a changelog and info and the info to the header of this PR.

File: src/config/ConfigService.js [hunk 3]
```diff
@@ -40,18 +40,13 @@ export default class ConfigService extends Service {
 
 
 	initialize() {
-		let config_url = this.App.Config.get('CONFIG_PATH');
-
-		// Provide backward compatibility with CONFIG_URL
-		if (this.App.Config.get('CONFIG_URL')) {
-			config_url = this.App.Config.get('CONFIG_URL');
-		}
+		const dynamic_config_url = document.getElementsByName('x-config')[0]?.content;
```
Comment (ateska) [inline src/config/ConfigService.js:43]: Can you add as a comment the example of the tag used for specification of the documentation? @titkov49 

Also this needs to be documented somewhere ...

Comment (titkov49) [reply src/config/ConfigService.js:43]: @ateska 
Done: https://github.com/TeskaLabs/asab-webui/pull/149

**Pull request merged**